addon/mixins/sl-tooltip-enabled.js

```diff
@@ -72,11 +68,10 @@ export default Ember.Mixin.create({
     /**
      * Enable popover
      *
-     * @private
```

Comment:
Seems this would still be applicable, denoting that it is a method used internal to the object to assist with other logic, but not intended to be called/executed directly.